fvtest/compilertriltest/BitPermuteTest.cpp

```diff
@@ -146,8 +147,8 @@ TEST_P(lBitPermuteTest, ConstAddressLengthTest)
       "  (lreturn                         "
       "  (lbitpermute                     "
       "   (lload parm=0)                  "
-      "   (aconst %llu)                   "
-      "   (iconst %llu)                   "
+      "   (aconst %p)                     "
```

Comment:
Why is `%p` used here instead of `OMR_PRIuPTR` with a cast, as is done bellow?

Reply:
Oops you're right, this should be printing in unsigned decimal (`PRIuPTR`), not `p` (which is usually hex).  By the way, does tril support hex integer constants?

Reply:
> By the way, does tril support hex integer constants?

It sure does! :smile: 

I think these tests used to use hex literals but got changed at some point with other things. I recall there being issues with signedness at one point, but that shouldn't affect address literals.

Reply:
Heh nice. I'm leaving the pointer formatted in decimal (using `OMR_PRIuPTR`) but that's good to know :+1: